inf-ruby.el

```diff
@@ -634,6 +634,10 @@ keymaps to bind `inf-ruby-switch-from-compilation' to `С-x C-q'."
 one of the predicates matches, then calls `inf-ruby-console-NAME',
 passing it the found directory.")
 
+(defvar inf-ruby-breakpoint-pattern "\\(\\[1\\] pry(\\)\\|\\((rdb:1)\\)"
+  "Pattern to check if the current line indicates the current compilation mode
```

Comment:
Please make the first line "a complete sentence that gives a summary description of the object". The following line(s) can be more verbose.